itou/job_applications/models.py

```diff
@@ -50,6 +57,7 @@ class JobApplicationWorkflow(xwf_models.Workflow):
         (STATE_NEW, "Nouvelle candidature"),
         (STATE_PROCESSING, "Candidature à l'étude"),
         (STATE_POSTPONED, "Candidature en liste d'attente"),
+        (STATE_PRIOR_TO_HIRE, "Candidature en pré-embauche"),
```

Comment:
Vu avec Marion. Le problème avec “Candidature en pré-embauche”, c’est que ce n’est pas la candidature qui est en pré-embauche mais le candidat. Comme nous avons déjà “Embauché ailleurs”, ou “Embauche annulée”, nous proposons d’unifier sur “Action préalable à l’embauche”.
```suggestion
        (STATE_PRIOR_TO_HIRE, "Action préalable à l’embauche"),
```